.travis.yml

```diff
@@ -173,7 +173,7 @@ jobs:
             - ccache
       env: >-
         MIXXX_ENVPATH="${HOME}/buildenv"
-        CMAKEFLAGS_EXTRA="-DCOREAUDIO=ON -DHSS1394=ON"
+        CMAKEFLAGS_EXTRA="-DCOREAUDIO=ON -DHSS1394=ON -DLOCALECOMPARE=OFF"
```

Comment:
Explicitly disabling the flag should not be necessary for the CMake build if this is the default setting? Let's keep the code with the workaround at a single place.

Reply:
Okay. I thought we wanted to make all build flags explicit in the CI configuration scripts.

Reply:
Only those that are supposed to be ON. This reduces clutter and has the desired effect of ensuring to build a version that is considered *feature complete*.